addons/website/views/snippets/s_searchbar.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_searchbar_input" name="Search">
+    <t t-call="website.website_search_box_input">
+        <t t-set="_classes" t-value="'s_searchbar_input'"/>
+        <t t-set="_snippet" t-value="'s_searchbar_input'"/>
+        <t t-set="search_type" t-value="'pages'"/>
+        <t t-set="action" t-value="'/pages'"/>
+        <t t-set="limit" t-value="'5'"/>
+        <t t-set="display_image" t-value="'true'"/>
+        <t t-set="display_description" t-value="'true'"/>
+        <t t-set="display_extra_link" t-value="'true'"/>
+        <t t-set="display_detail" t-value="'true'"/>
+    </t>
+</template>
+<template id="s_searchbar" name="Search">
+    <section class="s_searchbar bg-200 pt48 pb48">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 offset-lg-2">
+                    <h2>Search on our website</h2>
+                    <p>You will get results from blog posts, products, etc</p>
```

Comment:
Submitted to PO, might be taken into account for a future version of the search facility

Reply:
I would then rephrased the message, at least to add `pages` in it. Maybe also ideally be clear than it is one or another but not both? It rally sounds like it will do a global search.
I think I won't be the only one lost when trying it, I typed a blog name but nothing appeared, then a product name but nothing appeared.

Reply:
After all it seems the future version is now, and I have added a search through everything in https://github.com/odoo/odoo/pull/65871/commits/0bdbbad8f480627354d7b2444170c3f045bfcf1c

Reply:
:partying_face: _Future is the new present_